### ISSUE TITLE
KNOWN ISSUE: Display of quotations

### ISSUE THREAD

**Issue by wilkus27** [Connexions/cnx-recipes#669]
In **Astronomy** textbook sometimes occur quotations, which are written in many different ways. 

1. https://katalyst01.cnx.org/contents/LnN76Opl@14.4:x2ommVk0@3/10-1-The-Nearest-Planets-An-Overview

Here _blockquote_ is applied:

![poem1](https://user-images.githubusercontent.com/40228854/46865960-bfae9d80-ce1f-11e8-94fe-8e5f25518fb8.JPG)

But line breaks are inconsistent.

PDF:

![poem1_pdf](https://user-images.githubusercontent.com/40228854/46866031-08665680-ce20-11e8-9df1-e4c3d5453f10.JPG)

2. https://katalyst01.cnx.org/contents/LnN76Opl@14.4:Kw5Qt8rM@5/3-6-Gravity-with-More-Than-Two-Bodies

Here text is totally unreadable:

![poem2](https://user-images.githubusercontent.com/40228854/46866174-a1956d00-ce20-11e8-8796-5edeac7227fb.JPG)

PDF:

![poem2_pdf](https://user-images.githubusercontent.com/40228854/46866184-a78b4e00-ce20-11e8-844c-517d02c2fd9c.JPG)

**Comment by Adrianltd** [comment]
This is caused by the wrong content - the `div` elements are inside `p` elements which is not a valid html.
Then in the baking process the structure behaves strangely:
- the first line is being put into `p`, then there's `div` with `newline` class and then there's next line of poem (without container), then again `div`, again line of poem and so on.

In my opinion those poems should be made with one of two ways:
1. inside `<pre>` tags (then we need no `<br>`), eg.
> <pre>Lorem ipsum dolor sit amet,
>  consectetur adipiscing elit.
> Vestibulum non ornare nunc. 
> Pellentesque` libero justo</pre>
2. inside `<p>` tags with `<br>` after each line, eg.
> <p>Lorem ipsum dolor sit amet,<br>
> consectetur adipiscing elit. <br>
> Vestibulum non ornare nunc. <br>
> Pellentesque` libero justo</p>

**Comment by lwtchu** [comment]
The CNXML is inconsistently constructed. In the first instance, there's a `<para>` in a `<quote>` tag and in the second instance, it is the other way around. 

There are `<newline/>` tags between the lines to cause the line break in the PDF. I don't think there's any way around the `<newline/>` tag in CNXML. I don't think it's a huge problem for it to have double spacing on webview.  Bigger conversation: We might want to look into how the `<newline>` tag is being transformed.

In conclusion: 
The first instance - we CANNOT fix. It will have to look like that as is.
The second one: CMs will find `<quote>` nested in `<para>` and switch it around so it will at least have the bar running down the side on the left.
